.travis_scripts/build-release.sh

```diff
@@ -3,6 +3,11 @@
 set -e
 set -x
 
+# Set JAVA_HOME inside our CentOS Docker container.
+if [ -z "${JAVA_HOME}" -a -e /usr/lib/jvm/java-openjdk ] ; then
```

Comment:
How does this check interact with the OSX check below?

Reply:
As you can see, the OSX check only applies if JAVA_HOME is unset. Since this one would set JAVA_HOME, the two conditions are exclusive of each other.

Reply:
I figured that was the case. Thanks for the confirmation.